distribution/lib/Standard/Test/0.0.0-dev/src/Main.enso

```diff
@@ -166,7 +166,7 @@ specify : Text -> Any -> (Text | Nothing) -> Nothing
 specify label ~behavior pending=Nothing =
     pair = case pending of
         Nothing -> Duration.time_execution (run_spec behavior)
-        reason -> Pair_Data 0.milliseconds (Pending reason)
+        reason -> Pair_Data Duration.new (Pending reason)
```

Comment:
I'm wondering if it wouldn't be more readable if we made a `Duration.zero` alias which calls `new` with no arguments. Would be clearer that it is a 0-duration.